### ISSUE TITLE
fileServer doesn't set content-type header

### ISSUE THREAD

**Issue by justinwoo** [owickstrom/hyper#42]
for things like CSS files, the content-type needs to be written as text/css for Chrome to actually use it as a stylesheet, it seems.

**Comment by owickstrom** [comment]
All right. Having Hyper guess MIME types based on file extensions seems like something we'd regret later. Perhaps some lookup table that is user-configurable, like with Nginx *https://github.com/h5bp/server-configs-nginx/blob/master/mime.types)?

**Comment by justinwoo** [comment]
yeah, something like that would probably be best. like a default set and then the user could pass in their own record of how to handle stuff

**Comment by owickstrom** [comment]
I'm very tight on time currently. If you need it soon, would you like to send a PR? Thanks!

**Comment by justinwoo** [comment]
no rush! it's just on my vidtracker and i'm handling css requests separately for now

**Comment by paluh** [comment]
I've added simple lookup table for mime types: https://github.com/owickstrom/hyper/blob/master/src/Hyper/Node/FileServer.purs#L28

We can break API in next release and allow users to pass their own table too if that is not enough.

@justinwoo, @owickstrom Can I close it?

**Comment by owickstrom** [comment]
Looks like a good start @paluh. :+1: And yeah, let's make it a parameter when needed.
